<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_EditBook</class>
 <widget class="QDialog" name="Window_EditBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 201, 62);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_BookName">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 125, 160);
color: rgb(0, 135, 99);
border-color: rgb(255, 125, 160);</string>
       </property>
       <property name="placeholderText">
        <string> Book Name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_Search">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 145, 106);</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/img/img/search-icon.png</normaloff>:/img/img/search-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_BookId">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 125, 160);
border-color: rgb(255, 125, 160);
color: rgb(0, 135, 99);</string>
       </property>
       <property name="placeholderText">
        <string> Book ID    </string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_EditBook">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 145, 106);</string>
       </property>
       <property name="text">
        <string>Edit Book</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/img/img/rea.png</normaloff>:/img/img/rea.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_Subject">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 125, 160);
color: rgb(0, 135, 99);
border-color: rgb(255, 125, 160);</string>
       </property>
       <property name="placeholderText">
        <string> Subject    </string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_AuthorName">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 125, 160);
color: rgb(0, 135, 99);
border-color: rgb(255, 125, 160);</string>
       </property>
       <property name="placeholderText">
        <string> Author     </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/img/img/reedit.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_BookId">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string> Book ID    </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_BookName">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Book Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_AuthorName">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Author     </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_Subject">
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Subject </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
